Automattic/jetpack
# Jetpack Social: Add an image picker to the Jetpack Social editor panel

#### Changes proposed in this Pull Request:

This PR adds an image picker to the Jetpack Social panel when the enhanced publishing feature is enabled.

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






These steps are identical for Jetpack and Jetpack Social.
* Start writing a new post and check that the Jetpack Social panel looks the same as it did before.
* Add support for enhanced publishing to your site (context here: D90213-code)
* Refresh the page. You should now see an image picker (see screenshot).
* Pick an image, and make sure that you can delete and change it properly.
* Save as **draft**, refresh the page and check that the image got saved properly.
<img width="285" alt="image" src="https://user-images.githubusercontent.com/1713699/198008294-e1f73cbd-69d8-4847-ab40-4d9d1aab65ad.png">
